packages/frontend/app/components/reports/new-subject.gjs

```diff
@@ -447,6 +447,15 @@ export default class ReportsNewSubjectComponent extends Component {
         select.classList.add('error');
         select.focus();
       }
+
+      const searchObjectTypes = ['course', 'instructor', 'learning material', 'mesh term'];
+
+      if (searchObjectTypes.includes(this.prepositionalObject) && !this.prepositionalObjectId) {
+        const search = document.querySelector('[data-test-search-box] input');
```

Comment:
the targeted element is not guaranteed to be there, and will break the site when not.

running the report with this configuration will blow up, for example:

![image](https://github.com/user-attachments/assets/de60fc57-b69d-478e-90f6-820c21d55fd6)